harambe/__init__.py

```diff
@@ -11,4 +17,8 @@
     "AsyncScraperType",
     "AsyncScraper",
     "PAGE_PDF_FILENAME",
+    "HarambeException",
+    "SchemaValidationError",
+    "MissingRequiredFieldError",
```

Comment:
```suggestion
    "MissingRequiredFieldsError",
```

This should be plural. We will report all missing fields, not just one